app/cdap/components/Administration/TetheringTabContent/OdfTetheringConnections/PendingRequests/PendingRequestsTable.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+const TableContainer = styled.div``;
```

Comment:
I'm not going to comment on the rest of these because they must be intentional - why though?

Reply:
Ah sorry I should've probably added a comment about these divs. Most if not all will probably be used in my subsequent PRs when I add more child components, wanted to leave them there to keep the overall component structure. I can remove them and add them back if only necessary.